recipes/compute_slurm_config.rb

```diff
@@ -26,8 +26,8 @@
 
 # Mount /opt/slurm over NFS
 # Computemgtd config is under /opt/slurm/etc/pcluster; all compute nodes share a config
-mount '/opt/slurm' do
-  device(lazy { "#{node['cfncluster']['cfn_master_private_ip']}:/opt/slurm" })
+mount "#{node['cluster']['slurm']['install_dir']}" do
```

Comment:
`node['cluster']` should be `node['cfncluster']`, same for other places

Reply:
ok, I'll fix that now